stake-pool/js/src/index.ts

```diff
@@ -452,10 +452,10 @@ export async function withdrawStake(
 
     const availableForWithdrawal = calcLamportsWithdrawAmount(
       stakePool.account.data,
-      stakeAccount.lamports - MINIMUM_ACTIVE_STAKE - stakeAccountRentExemption,
+      new BN(stakeAccount.lamports - MINIMUM_ACTIVE_STAKE - stakeAccountRentExemption),
```

Comment:
how do we handle negative ints?

Reply:
Unless otherwise specified, I believe `BN` does signed numbers: https://github.com/indutny/bn.js/?tab=readme-ov-file#prefixes

And if you mean for this particular line, it shouldn't ever go negative since we're checking the existence of the account earlier, and the stake pool program guarantees the account has a certain minimum.

But I can add a defensive check to make sure it's positive here, if you like